src/styles/theme.scss

```diff
@@ -220,7 +258,25 @@ h5 {
 }
 .section--highlight {
   color: $text-colour;
-  background-color: $grey-light-purple;
+  background-color: $white-pure;
+}
+
+.section .section--story-end {
+  background-color: $white-pure;
+}
+
+.section--story-end :last-child {
+  align-self: flex-start;
+}
+
+.section--story-end .button[href*="tel"] {
+  background-color: transparent;
+  margin-bottom: 0;
+}
+
+.section--story-end .button[href*="tel"] span:last-of-type {
```

Comment:
Shouldn't need to do this - the phone cta is not as expected

Reply:
I am not clear on what you mean here, tbh. I am just changing the colour of the text as per the design. Are you saying that using `DesktopDisplayOnly` should take care of it without me having to do any of this?

Reply:
Yes that is what I am saying. It seems a little broken on Hospice Hope - but that ans Haven both use the phone CTA type in that same place.

I created a note to fix in the core.